h/views/api/users.py

```diff
@@ -21,19 +24,33 @@ def create(request):
     Client ID and Client Secret) to create users in their authority. These
     users are created pre-activated, and are unable to log in to the web
     service directly.
-    """
-    client = request_auth_client(request)
 
+    Note: the authority-enforcement logic herein is, by necessity, strange.
+    The API accepts an ``authority`` parameter but the only valid value for
+    the param is the client's verified authority. If the param is missing, the
+    client's authority will be used. If the param is present, but does not
+    match the client's authority, ``ValidationError`` is raised.
+
+    :raises ValidationError: if ``authority`` param does not match client
+                             authority
+    :raises ConflictError:   if user already exists
```

Comment:
I recently discovered that you can shorten these to `:raise` instead of `:raises` btw